packages/epo-react-lib/src/index.ts

```diff
@@ -1,10 +1,8 @@
 // styled-components
+export { tokens } from "@/styles/abstracts";
+export * from "@/styles/utils";
 
-export {
-  tokens,
-  fluidScale,
-  default as GlobalStyles,
-} from "@/styles/globalStyles";
+export { default as GlobalStyles } from "@/styles/globalStyles";
```

Comment:
now we export the tokens (including palettes), styling utilities, and global styles